tools/run_tests/run_python.sh

```diff
@@ -42,7 +42,12 @@ export LDFLAGS="-L$ROOT/libs/$CONFIG"
 export GRPC_PYTHON_BUILD_WITH_CYTHON=1
 export GRPC_PYTHON_ENABLE_CYTHON_TRACING=1
 
-tox
+if [ "$CONFIG" = "gcov" ]
```

Comment:
for code coverage we use the commands from tox.ini to run the tests and generate the coverage report.
For a "regular test run" (=opt or dbg), we are running tests in parallel and tox cannot handle that. So we make sure everything is built in advance and just run the test here.